lib/postgrex/protocol.ex

```diff
@@ -964,6 +1021,8 @@ defmodule Postgrex.Protocol do
     do: {:commit, nil}
   defp decode_tag("ROLLBACK"),
     do: {:rollback, nil}
+  defp decode_tag( :suspended ),
+    do: {:select, nil}
```

Comment:
We should use whatever tag that postgres is sending us and no custom logic. If we need to add more tag parsing to handle the code lets do that.